test/sanity/phases/smoke-tests.sh

```
@@ -0,0 +1,238 @@
+  cd "$WORKDIR"
+  altimate_run "discover-mcps" --command discover-and-add-mcps "list"
+  local output=$(get_output "discover-mcps")
+  if echo "$output" | grep -qi "command not found\|Unknown command"; then
+    echo "FAIL" > "$RESULTS_DIR/discover-mcps"
+  else
+    echo "PASS" > "$RESULTS_DIR/discover-mcps"
+  fi
+}
+
+test_configure_claude() {
+  cd "$WORKDIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)